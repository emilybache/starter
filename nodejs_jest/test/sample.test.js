
const sample = require('../src/sample.js');

describe('Sample', () => {

    it('should do something', () => {
        
        let result = sample()
        expect(result).toMatchSnapshot()
    })
});