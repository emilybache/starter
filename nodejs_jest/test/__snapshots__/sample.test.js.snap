// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample should do something 1`] = `"Change Me"`;
